assets/js/cart.js

```diff
@@ -0,0 +1,108 @@
+    { id: 6, name: 'fff', price: 425, category: 'clothes', URL: './assets/images/7.jpg', detail: 'This product from fashion store' },
+    { id: 7, name: 'ggg', price: 170, category: 'clothes', URL: './assets/images/8.jpg', detail: 'This product from fashion store' }
+]
+
+let cards = document.getElementById("cart-products");
+//localStorage.setItem("my_products", JSON.stringify(products));
+
+
+showProducts();
+function showProducts() {
+
+    cards.innerHTML = "";
+    products.forEach(function (product, i) {
+
+        let card = document.createElement("div");
+        let cardHeaderInfo = document.createElement("div");
+
+        let productImage = document.createElement("img");
+        productImage.src = product.URL;
+        let productPrice = document.createElement("h3");
+        productPrice.textContent = product.price;
+        let productCategory = document.createElement("h3");
+        productCategory.textContent = product.category;
+        let productName = document.createElement("h2");
+        productName.textContent = product.name;
+        let dolarsign = document.createElement("span");
+        dolarsign.textContent = ' $';
+        let productDetail = document.createElement("p");
+        productDetail.textContent = product.detail;
+        let deleteBtn = document.createElement("button");
+        let deleteIcon = document.createElement("i");
+        deleteIcon.className = 'fa fa-trash deleteIcon';
+        deleteBtn.appendChild(deleteIcon);
+
+        cardHeaderInfo.appendChild(productImage);
+        cardHeaderInfo.appendChild(productCategory);
+        productPrice.appendChild(dolarsign);
+        cardHeaderInfo.appendChild(productPrice);
+        card.appendChild(cardHeaderInfo);
+        card.appendChild(productName);
+        card.appendChild(productDetail);
+        card.appendChild(deleteBtn);
+        cards.appendChild(card);
+
+
+        card.className = 'cardItem';
+        cardHeaderInfo.className = 'cardHeaderInfo';
+        productImage.className = 'cardImg';
+        productCategory.className = 'cardCategory';
+        productPrice.className = 'cardPrice';
+        productName.className = 'cardName';
+        productDetail.className= 'cardDetail';
+        deleteBtn.id = 'cartDeleteBtn';
+
+
+        deleteBtn.addEventListener("click", function() {
+            let confirmDelete = confirm("Are you sure you want to delete this product from your cart?");
+        
+            if (confirmDelete) {
+                products.splice(i,1);
+                showProducts();
+                sum=0;
+                calcSum();
+            }
+            
+        });
+        
+
+
+    }
+    );
+
+
+
+    
+}
+
+let cartSum = document.getElementById("cartSum");
+let cartTax = document.getElementById("cartTax");
+let cartTotlaSum = document.getElementById("cartTotalSum");
+let sum = 0, tax,totalSum;
```

Comment:
I think these need to be moved inside of `calcSum`, so that they are reinitialized every time calcSum is called.  Otherwise, sum will continue to increase whenever calcSum is called again, instead of starting from 0 each time.